src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,22 @@
+package duke.command;
+
+import duke.Storage;
+import duke.Ui;
+import duke.task.Task;
+import duke.task.TaskList;
+
+public class DeleteCommand extends Command {
+    public DeleteCommand(String arguments) {
+        super(arguments);
+    }
+
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
+        int index = Integer.parseInt(this.arguments);
+        Task deletedTask = tasks.get(index);
+        tasks.remove(index);
```

Comment:
not a coding standard violation, but `List.remove()` returns a `Task`, so you can actually do Task deletedTask = tasks.remove(index) instead :)